tests/_utils/view.js

```diff
@@ -24,7 +24,62 @@ const TEXT_RANGE_START_TOKEN = '{';
 const TEXT_RANGE_END_TOKEN = '}';
 
 /**
- * Converts view elements to its HTML-like string representation.
+ * Writes the contents of the {@link engine.treeView.TreeView TreeView} to an HTML-like string.
+ *
+ * @param {engine.treeView.TreeView} treeView
+ * @param {Object} [options]
+ * @param {Boolean} [options.withSelection=false] Whether to write the selection.
+ * @param {Boolean} [options.rootName='main'] Name of the root from which data should be stringified. If not provided
+ * default `main` name will be used.
+ * @param {Boolean} [options.showType=false] When set to `true` type of elements will be printed (`<container:p>`
+ * instead of `<p>` and `<attribute:b>` instead of `<b>`).
+ * @param {Boolean} [options.showPriority=false] When set to `true` AttributeElement's priority will be printed
+ * (`<span:12>`, `<b:10>`).
+ * @returns {String} The stringified data.
+ */
+export function getData( treeView, options = {} ) {
+	const withSelection = !!options.withSelection;
+	const rootName = options.rootName || 'main';
+	const root = treeView.getRoot( rootName );
+	const stringifyOptions = {
+		showType: options.showType,
+		showPriority: options.showPriority,
+		ignoreRoot: true
+	};
+
+	return withSelection ?
+		getData._stringify( root, treeView.selection, stringifyOptions ) :
+		getData._stringify( root, null, stringifyOptions );
+}
+
+// Set stringify as getData private method - needed for testing/spying.
+getData._stringify = stringify;
+
+/**
+ * Sets the contents of the {@link engine.treeView.TreeView TreeView} provided as HTML-like string.
+ *
+ * @param {engine.treeView.TreeView} treeView
+ * @param {String} data HTML-like string to write into TreeView.
+ * @param {Object} options
+ * @param {String} [rootName] Root name where parsed data will be stored. If not provided, default `main` name will be
+ * used.
+ */
+export function setData( treeView, data, options = {} ) {
+	const rootName = options.rootName || 'main';
+	const root = treeView.getRoot( rootName );
+	root.removeChildren( 0, root.getChildCount() );
+	const result = setData._parse( data, { rootElement: root } );
```

Comment:
You can use destructuring:
`const { view, selection } = setData._parse( data, { rootElement: root } );`